Core/CMP API Specification.md

```diff
@@ -176,22 +176,113 @@ cmpDisplayStatus: String, // possible values: hidden, visible, disabled
+// List of supported APIs (section ids and prefix strings), e.g. used while loading.
+// Example: ["2:tcfeuv2","6:uspv1"] 
+supportedAPIs : Array of string,
 
-cmpId : Number, // IAB assigned CMP ID, may be 0 during stub/loading. Reference the above CMP ID section for additional information.
+// IAB assigned CMP ID, may be 0 during stub/loading. Refer the above CMP ID section for additional information.
+cmpId : Number,
 
 sectionList : Array of Number, // may be empty during loading of the CMP
 
-applicableSections: Array of Number, // Section ID considered to be in force for this transaction. In most cases, this field should have a single section ID. In rare occasions where such a single section ID can not be determined, the field may contain up to 2 values. During the transition period which ends on September 30, 2023, the legacy USPrivacy section may be determined as applicable along with another US section. In this case, the field may contain up to 3 values where one of the values is 6, representing the legacy USPrivacy section. The value can be 0 or a Section ID specified by the Publisher / Advertiser, during stub / load. When no section is applicable, the value will be [-1].
+// Section ID considered to be in force for this transaction.
+// In most cases, this field should have a single section ID. In rare occasions where such a single section ID
+// can not be determined, the field may contain up to 2 values. During the transition period which ends on
+// September 30, 2023, the legacy USPrivacy section may be determined as applicable along with another US section.
+// In this case, the field may contain up to 3 values where one of the values is 6, representing the
+// legacy USPrivacy section. The value can be 0 or a Section ID specified by the Publisher / Advertiser, during
+// stub / load.
+// When no section is applicable, the value will be [-1].
+applicableSections: Array of Number,
 
 gppString: String // the complete encoded GPP string, may be empty during CMP load
 
-parsedSections: Object // The parsedSections property represents an object of all parsed sections of the gppString property that are supported by the API on this page (see supportedAPIs property). The object contains one property for each supported API with the name of the API as the property name and the value as a parsed representation of this section (similar to getSection command). If a section is supported but not represented in the gppString, it is omitted in the parsedSections object.
+// The parsedSections property represents an object of all parsed sections of the gppString property that are supported
+// by the API on this page (see supportedAPIs property). The object contains one property for each supported API with
+// the name of the API as the property name and the value as a parsed representation of this section (similar to
```

Comment:
let's get rid of this siilar to wording and adopt the wording in #133 or some evolution of it.